explainaboard/metrics/qa_table_text_hybrid.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import annotations
+
+import abc
+from dataclasses import dataclass
+from typing import Optional
+
+import numpy as np
+
+from explainaboard.metrics.metric import (
+    Metric,
+    MetricConfig,
+    MetricStats,
+    SimpleMetricStats,
+)
+from explainaboard.metrics.registry import register_metric_config
+from explainaboard.utils.eval_utils import qa_table_text_hybrid as eval_util
+from explainaboard.utils.preprocessor import ExtractiveQAPreprocessor, Preprocessor
+from explainaboard.utils.typing_utils import unwrap_or
+
+
+class HybridQAMetric(Metric):
+    """
+    An abstract class for HybridQA tasks that measures scores after normalization.
```

Comment:
I'm not super-familiar with the idea of a HybridQA task and presumably other people reading this code might not be either. Maybe you should provide a link to the relevant processor, and in that processor add a few more references/documentation about what kinds of tasks they are.